app/src/main/java/movierating/services/ReviewService.java

```diff
@@ -1,9 +1,11 @@
 package movierating.services;
 
+import movierating.models.Movie;
+import movierating.models.Review;
 import movierating.repositories.MovieRepository;
 import movierating.repositories.UserRepository;
 
 public interface ReviewService {
 
-    public void addReview();
+    public void addReview(String[] fields);
```

Comment:
Shouldn't this be `public void addReview(String userName, String movieName, int rating);` ?
Because the interface we choose here must be self explanatory about the arguments it takes. It should not worry about the input format.

Reply:
Changed this.